### PR TITLE
extend HVU data tools

### PR DESCRIPTION
**Pull request by kennymckormick**
Extend HVU data tools.
Now it can generate data file list for a specific tag category, which is the prerequisite of HVU action baseline releasing.

### CODE REVIEW

File: tools/data/hvu/preparing_hvu.md [hunk 1]
```diff
@@ -43,15 +43,43 @@ More details can be found in [data_preparation](/docs/data_preparation.md)
...
 # execute the command below when rawframes are ready
 bash generate_rawframes_filelist.sh
 ```
 
-## Step 5. Folder Structure
+## Step 5. Generate File List for each individual tag categories.
+
+This part is **optional** if you don't want to train models on HVU for a specific tag category.
+
+Thhe file list generated in step 4 contain labels of different categories. These file lists can only be
```
Comment (dreamerlin) [inline tools/data/hvu/preparing_hvu.md:58]: contains

File: tools/data/hvu/generate_sub_file_list.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        label = item['label']
+        if category in label:
+            item['label'] = label[category]
+            return item
+        else:
+            return None
+
+    data = [parse_item(item, category) for item in data]
+    data = [item for item in data if item is not None]
+    mmcv.dump(data, target_file)
```
Comment (dreamerlin) [inline tools/data/hvu/generate_sub_file_list.py:29]:
```python
result = []
for item in data:
    label = item['label']
    if category in label:
        item['label'] = label[category]
        result.append(item)
mmcv.dump(result, target_file)
```

**Review by dreamerlin — APPROVED** [review]

**Review by SuX97 — COMMENTED** [review]
LGTM.

**Review by SuX97 — APPROVED** [review]

File: tools/data/hvu/preparing_hvu.md [hunk 3]
```diff
@@ -43,15 +43,43 @@ More details can be found in [data_preparation](/docs/data_preparation.md)
...
+You can run the follow scripts to generate file list in the format of videos and rawframes, respectively.
 
 ```shell
 bash generate_videos_filelist.sh
 # execute the command below when rawframes are ready
 bash generate_rawframes_filelist.sh
 ```
 
-## Step 5. Folder Structure
+## Step 5. Generate File List for each individual tag categories.
```
Comment (innerlee) [inline tools/data/hvu/preparing_hvu.md:54]: capitalization

**Review by innerlee — APPROVED** [review]

File: tools/data/hvu/preparing_hvu.md (hunk 3 above)
Comment (kennymckormick) [reply tools/data/hvu/preparing_hvu.md]: All 1st letters are capitalized except the conjunction `for`

**Pull request merged**